### PR TITLE
Go: Use LRO client

### PR DESCRIPTION
**Pull request by pongad**
This change only affects Go. There are some baseline changes in other languages due to the added `GetBigBook` method in `library.proto`.

### CODE REVIEW

File: src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java [hunk 1]
```diff
@@ -247,9 +255,13 @@ static ModelTypeTable createTypeTable() {
 
   @VisibleForTesting
   List<StaticLangApiMethodView> generateApiMethods(
-      SurfaceTransformerContext context, List<Method> methods, List<String> pageStreamImports) {
+      SurfaceTransformerContext context,
+      List<Method> methods,
+      List<String> pageStreamImports,
```
Comment (jba) [inline src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java]: Maybe a more general map from enum to List<String>? I suspect there will be more of these.

Comment (pongad) [inline src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java]: This is a good point. I'll look into this.

**Comment by pongad** [comment]
@jba @garrettjonesgoogle PTAL

**Comment by jba** [comment]
LGTM.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**